x-pack/plugins/maps/public/connected_components/map_settings_panel/spatial_filters_panel.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiPanel, EuiSpacer, EuiSwitch, EuiSwitchEvent, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { MapSettings } from '../../reducers/map';
+import { AlphaSlider } from '../../components/alpha_slider';
+import { MbValidatedColorPicker } from '../../layers/styles/vector/components/color/mb_validated_color_picker';
+
+interface Props {
+  settings: MapSettings;
+  updateMapSetting: (settingKey: string, settingValue: string | number | boolean) => void;
+}
+
+export function SpatialFiltersPanel({ settings, updateMapSetting }: Props) {
+  const onAlphaChange = (alpha: number) => {
+    updateMapSetting('spatialFiltersAlpa', alpha);
+  };
+
+  const onFillColorChange = (color: string) => {
+    updateMapSetting('spatialFiltersFillColor', color);
+  };
+
+  const onLineColorChange = (color: string) => {
+    updateMapSetting('spatialFiltersLineColor', color);
+  };
+
+  const onShowSpatialFiltersChange = (event: EuiSwitchEvent) => {
+    updateMapSetting('showSpatialFilters', event.target.checked);
+  };
+
+  const renderStyleInputs = () => {
+    if (!settings.showSpatialFilters) {
+      return null;
+    }
+
+    return (
+      <>
+        <AlphaSlider alpha={settings.spatialFiltersAlpa} onChange={onAlphaChange} />
+
+        <EuiFormRow
+          label={i18n.translate('xpack.maps.mapSettingsPanel.spatialFiltersFillColorLabel', {
+            defaultMessage: 'Fill color',
+          })}
+          display="columnCompressed"
+        >
+          <MbValidatedColorPicker
```

Comment:
consider initializing with this orange:

`#DA8B45`

it works well with differnet base-styles:

![image](https://user-images.githubusercontent.com/1833023/80022455-159ae180-84aa-11ea-85d5-81df4ead1278.png)

![image](https://user-images.githubusercontent.com/1833023/80022498-29464800-84aa-11ea-9f24-5e8a28d55911.png)